android/app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,9 @@
               android:name="io.flutter.embedding.android.NormalTheme"
               android:resource="@style/NormalTheme"
               />
+            <meta-data
```

Comment:
This <meta-data> needs to be outside the <activity> but inside the <application> tag. Like here: https://github.com/googleads/googleads-mobile-flutter/blob/master/packages/google_mobile_ads/example/android/app/src/main/AndroidManifest.xml